app/i18n/en/gen.php

```diff
@@ -79,6 +79,7 @@
 		'last_year' => 'Last year',
 		'mar' => 'mar',
 		'march' => 'Mar',
+		'may_' => 'May',
```

Comment:
Note that for the English localization this May should be lowercase for consistency with the rest, although I think it'd be better to change the others to `'mar' => 'Mar',` etc.